publish_pods.sh

```diff
@@ -0,0 +1,21 @@
+pod trunk push Core/StitchCoreSDK/StitchCoreSDK.podspec --allow-warnings --verbose
```

Comment:
[minor] if my docs stuff goes in before this, add `contrib/publish_docs.sh` before pushing any of the pods. that script will generate and publish the docs as long as you have the right AWS keys set up in your env